scripts/presubmit-tests.sh

```diff
@@ -296,6 +296,8 @@ function main() {
     kubectl version --client
     echo ">> go version"
     go version
+    echo ">> ko version"
+    ko version
```

Comment:
this command somehow doesn't work on my machine

Reply:
You must use the latest ko.